lib/server.js

```diff
@@ -108,8 +108,8 @@ Server.prototype._publish = function() {
   var bridgeConfig = this._config.bridge || {};
 
   var info = this._bridge.getService(Service.AccessoryInformation);
-  info.setCharacteristic(Characteristic.Manufacturer, `${toTitleCase(require('../package.json').author.name)}`);
-  info.setCharacteristic(Characteristic.Model, `${toTitleCase(require('../package.json').name)}`);
+  info.setCharacteristic(Characteristic.Manufacturer, bridgeConfig.manufacturer || `${toLowerCase(require('../package.json').author.name)}`);
```

Comment:
Did you want to make the default `homebridge.io`?

```js
  info.setCharacteristic(Characteristic.Manufacturer, bridgeConfig.manufacturer || 'homebridge.io'`);
```

Reply:
I updated it for this!